pkg/cmd/server/kubernetes/node/node.go

```diff
@@ -4,34 +4,29 @@ import (
 	"errors"
 	"fmt"
 	"net"
-	"net/url"
 	"os"
 	"path/filepath"
 	"time"
 
-	dockerclient "github.com/fsouza/go-dockerclient"
 	"github.com/golang/glog"
 
 	kapiv1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	kclientsetexternal "k8s.io/client-go/kubernetes"
-	"k8s.io/kubernetes/pkg/kubelet/dockershim"
-	dockertools "k8s.io/kubernetes/pkg/kubelet/dockershim/libdocker"
-	"k8s.io/kubernetes/pkg/util/mount"
+	"k8s.io/kubernetes/cmd/kubelet/app"
 	"k8s.io/kubernetes/pkg/volume"
 
 	configapi "github.com/openshift/origin/pkg/cmd/server/api"
 	cmdutil "github.com/openshift/origin/pkg/cmd/util"
-	dockerutil "github.com/openshift/origin/pkg/cmd/util/docker"
 	"github.com/openshift/origin/pkg/volume/emptydir"
 )
 
-const minimumDockerAPIVersionWithPullByID = "1.22"
```

Comment:
we don't check for the docker version anymore? or is this part of upstream

Reply:
> we don't check for the docker version anymore? or is this part of upstream

If we need it, we'll need need to move it upstream.  I'm 95% sure that our RPMS and ansible ensure the prereq level already.  @sdodson

Reply:
I'd be surprised if upstream didn't check that as well.

Reply:
And here you have it: https://github.com/openshift/origin/blob/8b81ebdcc7c1c70c05f12fd4e47a7d85ad9117fe/vendor/k8s.io/kubernetes/pkg/kubelet/cm/container_manager_linux.go#L708